library/VTKExtensions/Readers/Testing/TestF3DAssimpImportError.cxx

```diff
@@ -0,0 +1,64 @@
+#include <vtkNew.h>
+#include <vtkTestUtilities.h>
+
+#include "vtkCommand.h"
+#include "vtkF3DAssimpImporter.h"
+
+#include <iostream>
+#include <vector>
+
+class WarningEventCallback : public vtkCommand
+{
+public:
+  static WarningEventCallback* New() { return new WarningEventCallback; }
+
+  void Execute(vtkObject* caller, unsigned long evId, void* data) override
+  {
+    auto importer = reinterpret_cast<vtkF3DAssimpImporter*>(caller);
+    char* message = static_cast<char*>(data);
+    if (importer && message)
+    {
+      this->Messages.push_back(message);
+    }
+  }
+
+  const std::vector<std::string>& GetRecordedWarningMessages() const { return this->Messages; }
+
+private:
+  std::vector<std::string> Messages;
+};
+
+int TestF3DAssimpImportError(int argc, char* argv[])
+{
+  vtkObject::GlobalWarningDisplayOn();
+  vtkNew<vtkF3DAssimpImporter> importer;
+
+  vtkNew<WarningEventCallback> warningEventCallback;
+  importer->AddObserver(vtkCommand::WarningEvent, warningEventCallback);
+
+  importer->SetFileName("dummy.cae");
```

Comment:
is ".cae" an actual extension or is this a typo ?

Reply:
This is a typo, the correct extension is `.dae`. I've just fixed it.